refm/api/src/_builtin/Range

```diff
@@ -105,7 +105,7 @@ Range#cover? は連続値を扱います。
   
   # Date, DateTime の例
   (Date.new(2014,1,3)..Date.new(2014,1,5)).include?(Date.new(2014,1,5))           # => true
-  (Time.new(2014,1,3)..Time.new(2014,1,5)).include?(Time.new(2014,1,4,10,10,10))  # => "can't iterate from Time"
```

Comment:
bugsを確認すると以下のチケットで挙動が変わったみたいですね。
[Bug #11113: Time ranges cannot be used in case statements in 1.9+ but they could in 1.8.7 - Ruby trunk - Ruby Issue Tracking System](https://bugs.ruby-lang.org/issues/11113)

2.1.6/2.2.2ではこのエラーになりますが、2.1.7、2.2.3からはエラーにならなさそうです。

```
% ruby -ve '(Time.new(2014,1,3)..Time.new(2014,1,5)).include?(Time.new(2014,1,4,10,10,10))'
ruby 2.1.6p336 (2015-04-13 revision 50298) [x86_64-darwin16.0]
-e:1:in `each': can't iterate from Time (TypeError)
	from -e:1:in `include?'
	from -e:1:in `include?'
	from -e:1:in `<main>'
% ruby -ve '(Time.new(2014,1,3)..Time.new(2014,1,5)).include?(Time.new(2014,1,4,10,10,10))'
ruby 2.1.7p400 (2015-08-18 revision 51632) [x86_64-darwin14.0]
% ruby -ve '(Time.new(2014,1,3)..Time.new(2014,1,5)).include?(Time.new(2014,1,4,10,10,10))'
ruby 2.2.2p95 (2015-04-13 revision 50295) [x86_64-darwin14]
-e:1:in `each': can't iterate from Time (TypeError)
    from -e:1:in `include?'
    from -e:1:in `include?'
    from -e:1:in `<main>'
% ruby -ve '(Time.new(2014,1,3)..Time.new(2014,1,5)).include?(Time.new(2014,1,4,10,10,10))'
ruby 2.2.3p173 (2015-08-18 revision 51636) [x86_64-darwin14]
% ruby -ve '(Time.new(2014,1,3)..Time.new(2014,1,5)).include?(Time.new(2014,1,4,10,10,10))'
ruby 2.3.3p222 (2016-11-21 revision 56859) [x86_64-darwin16]
```

コミット的にはこのあたり
2.1: https://github.com/ruby/ruby/commit/7a03d4637cefadadad31fff5bdd6c20a8e5a00bd
2.2: https://github.com/ruby/ruby/commit/3cc321812808532a086c6020bb0c92dff1c0732c